sonatype-ci-release/src/main/scala/org/typelevel/sbt/TypelevelSonatypeCiReleasePlugin.scala

```diff
@@ -57,9 +62,52 @@ object TypelevelSonatypeCiReleasePlugin extends AutoPlugin {
 
       tags ++ branches
     },
+    tlCiReleaseStepSummaryTableInfo := {
+      val map: Map[String, String] = Map("Release version" -> (ThisBuild / version).value)
+      (ThisBuild / apiURL).value.map(_.toString).fold(map)(r => map + ("Api URL" -> r))
+    },
+    tlCiAddResolverInfoToSummary := true,
     githubWorkflowTargetTags += "v*",
     githubWorkflowPublish := Seq(
       WorkflowStep.Sbt(List("tlRelease"), name = Some("Publish"))
     )
   )
+
+  private def renderSummaryTable(results: Map[String, String]): String =
+    results
+      .toList
+      .map { case (k, v) => s"| ${k} | ${v} |" }
+      .mkString(s"| Build Result | Value |\n| -: | :- |\n", "\n", "\n\n")
+
+  override def projectSettings: Seq[Setting[_]] = Seq(
+    publish := {
+      val table: Map[String, String] = tlCiReleaseStepSummaryTableInfo.value
+      val projectName: String = name.value
+      val maybeMavenResolverUrl: Option[(String, String)] =
+        (ThisBuild / publishTo).value.collect {
+          case x: MavenRepo => (x.name, x.root)
+          case x: MavenRepository => (x.name, x.root)
+        }
+
+      val header: String = s"### ${projectName} Publication Summary\n"
+
+      val textToRender: String =
+        maybeMavenResolverUrl
+          .filter(_ => tlCiAddResolverInfoToSummary.value)
+          .fold(renderSummaryTable(table)) {
+            case (n, u) =>
+              val newTable: Map[String, String] = table + ("Resolver" -> s""""${n}" -> ${u}""")
+
+              val instructions: String =
+                s"To configure your build to use this published version set\n" +
+                  s"""`resolvers += Resolver.url("${n}", url("${u}"))`\n\n"""
+
+              renderSummaryTable(newTable) + instructions
+          }
+
+      GitHubActionsPlugin.appendtoStepSummary(header + textToRender)
+
+      publish.value
```

Comment:
Hm, should we run this first, before populating the table with the results? You know, in case it fails 😂

Reply:
<you_saw_nothing_madagascar_meme.png> here